SPECS/qt5-qtbase/qt5-qtbase.spec

```diff
@@ -33,7 +33,7 @@
 Name:         qt5-qtbase
 Summary:      Qt5 - QtBase components
 Version:      5.12.11
-Release:      5%{?dist}
+Release:      6%{?dist}
```

Comment:
In addition to bumping the release version, you will also need to add a `%patchX: CVE-2023-24607` macro for the patch itself and to make sure that it applies.

Reply:
Closing this, I didn't realize this was an exisitng patch that needed tweaking and was present below as Patch81